test/common/tests.js

```diff
@@ -1315,9 +1315,21 @@ module.exports = (sql, driver) => {
       })
     },
 
+    'force ODBC driver verify' (cfg) {
+      assert.ok(cfg.conn_str.startsWith('Driver=ODBC Driver 17 for SQL Server'))
+    },
+
+    'force ODBC driver query' (done) {
+      sql.query('select serverproperty(N\'servername\') as servername; ', (err, result) => {
+        assert.ok(result.recordset[0].servername)
```

Comment:
If this rejects does the test fail? I think the assertion needs to be wrapped in a try catch and the error passed to done().

Using promises is a little easier:

```js
sql.query('select serverproperty(N\'servername\') as servername; ').then(() => {
  assert.ok(result.recordset[0].servername)
  done();
}).catch(done);
```

vs

```js
sql.query('select serverproperty(N\'servername\') as servername; ', (err, result) => {
  if (err) {
    done(err);
  } else {
    try {
      assert.ok(result.recordset[0].servername)
      done()
    } catch (e) {
      done(e)
    }
  }
});
```

Reply:
Technically it would fail in the before statement when it tried (and failed) to connect.

Maybe all I need is a test of the forced connection and leave it at that.
```
    ...
    'connection healthy works force ODBC' (config, done) {
      config.options.driver = 'ODBC Driver 17 for SQL Server'
      this['connection healthy works'](config, done)
    },
    ...
```